autoload/limelight.vim

```diff
@@ -139,7 +139,7 @@ function! s:dim(coeff)
   let fg = synIDattr(synid, 'fg#')
   let bg = synIDattr(synid, 'bg#')
 
-  if has('gui_running') || exists('+termguicolors') || has('nvim') && $NVIM_TUI_ENABLE_TRUE_COLOR
+  if has('gui_running') || (exists('+termguicolors') && termguicolors)|| has('nvim') && $NVIM_TUI_ENABLE_TRUE_COLOR
```

Comment:
I looked it up, but the conversation wasn't about `has()` vs. `exists()`. `has()` has long been a conventional way of checking the existence of a feature. "If it ain't broke, don't fix it."